### PR TITLE
[INLONG-4972][TubeMQ] Add a command-line tool for TubeMQ 

### LINKED ISSUE #4972: [Feature] Add TubeMQ Command Tools
### Description

like `tubectl`, we can create/delete topic, produce/consume message.

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fancycoderzf**
### Prepare a Pull Request

- Fixes #4972 

### Motivation

Add a command line tool to facilitate system users to manage TubeMQ through the command line.

### Modifications

Wrote a Tubectl tool that supports 
* topic query, addition, modification,  and deletion;
* messages production and consumption;
* consumer group management.

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? yes
  - If yes, how is the feature documented? docs

### CODE REVIEW

**Comment by dockerzhang** [comment]
Please run `mvn spotless:apply` to fix the checkstyle issues.

**Comment by fancycoderzf** [comment]
I have run successfully `mvn spotless:apply` locally.

File: inlong-tubemq/pom.xml [hunk 1]
```diff
@@ -41,6 +41,7 @@
 
     <properties>
         <inlong.root.dir>${project.parent.basedir}</inlong.root.dir>
+        <jcommander.version>1.78</jcommander.version>
```
Comment (dockerzhang) [inline inlong-tubemq/pom.xml:44]: There is `jcommander.version` definition in the pom.xml, and you could use it directly.

Comment (fancycoderzf) [reply inlong-tubemq/pom.xml:44]: I will delete this line soon.

File: licenses/inlong-tubemq-server/licenses/LICENSE-jcommander.txt
Comment (gosonzhang) [inline licenses/inlong-tubemq-server/licenses/LICENSE-jcommander.txt:1]: The dependent package's name, version, and other information need to be registered in the LICENSE index, see :https://github.com/apache/inlong/blob/master/licenses/inlong-tubemq-server/LICENSE#L419

File: inlong-tubemq/tubemq-server/pom.xml [hunk 2]
```diff
@@ -187,6 +187,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.beust</groupId>
+            <artifactId>jcommander</artifactId>
+            <version>${jcommander.version}</version>
```
[the remark this answers is not in the document]
Comment (fancycoderzf) [reply inlong-tubemq/tubemq-server/pom.xml]: I have omitted that content.

**Review by gosonzhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**